code/stable/dblpendulum/SRS/JSON/DblPendulum_SRS.ipynb

```diff
@@ -167,7 +167,7 @@
     "## Organization of Document\n",
     "<a id=\"Sec:DocOrg\"></a>\n",
     "\n",
-    "The organization of this document follows the template for an SRS for scientific computing software proposed by <a href=#koothoor2013>koothoor2013</a>  and <a href=#smithLai2005>smithLai2005</a> . The presentation follows the standard pattern of presenting goals, theories, definitions, and assumptions. For readers that would like a more bottom up approach, they can start reading the [instance models](#Sec:IMs) and trace back to find any additional information they require.\n",
+    "The organization of this document follows the template for an SRS for scientific computing software proposed by <a href=#koothoor2013>koothoor2013</a> , <a href=#smithLai2005>smithLai2005</a> , <a href=#smithEtAl2007>smithEtAl2007</a>  and <a href=#smithKoothoor2016>smithKoothoor2016</a> . The presentation follows the standard pattern of presenting goals, theories, definitions, and assumptions. For readers that would like a more bottom up approach, they can start reading the [instance models](#Sec:IMs) and trace back to find any additional information they require.\n",
```

Comment:
It looks like there are extra spaces being added after the references for Jupyter notebooks; if this isn't already a <kbd>[newcomers](https://github.com/JacquesCarette/Drasil/labels/newcomers)</kbd> issue, it should be!